### PR TITLE
Bootstrap the elections

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Comment by daenney** [comment]
👍

File: share/elections/2016-01.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* Julien Pivotto (roidelapluie@inuits.eu)
+  * Accepting candidatures
+  * Communication before the election
+* Daniele Sluijters
+  * Voting process
+  * Encouraging people to vote
+  * Communication for the election
+  * Announcement of the results
+
+According to our rules, the Election Officers can not run in the Election.
```
Comment (tuxmea) [inline share/elections/2016-01.md]: Does that mean that an Election Officer can not be proposed as Candidate?

Comment (roidelapluie) [inline share/elections/2016-01.md]: yes, that is in the governance document https://github.com/voxpupuli/plumbing/blob/master/share/governance.md

> 1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating

**Comment by nibalizer** [comment]
Well this is super neat! :+1:

**Pull request merged**